test/ERC404.t.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../src/contracts/examples/ExampleERC404.sol";
+
+contract Erc404Test is Test {
+    ExampleERC404 public pandora;
```

Comment:
Remove pandora references here, lets keep it as close to generalized as possible